src/test/java/org/fogbowcloud/manager/core/processors/FulfilledProcessorTest.java

```diff
@@ -99,355 +95,365 @@ public void tearDown() {
         super.tearDown();
     }
 
-    @SuppressWarnings("unchecked")
-    @Test
-    public void testRunProcessWithExceededConnectionAttempts() throws InterruptedException, UnexpectedException, NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
-        Order order = this.createOrder();
-        order.setOrderState(OrderState.FULFILLED);
-         
-        this.fulfilledProcessor = new FulfilledProcessor(BaseUnitTests.LOCAL_MEMBER_ID,
-                this.tunnelingService, this.sshConnectivity,
-                DefaultConfigurationConstants.FULFILLED_ORDERS_SLEEP_TIME);
-
-        Field declaredField = this.fulfilledProcessor.getClass().getDeclaredField("connectionAttempts");
-        declaredField.setAccessible(true);
-                
-        this.connectionAttempts = (Map<String, Integer>) declaredField.get(this.fulfilledProcessor);
-        
-        for (int i = 0; i <= 1000; i++) {
-            connectionAttempts.put(createOrder().getId(), i);
-        }
-              
-        declaredField.set(this.fulfilledProcessor, this.connectionAttempts);
-        
-        Assert.assertEquals(this.connectionAttempts.size(), 1001);
-        
-        this.thread = new Thread(this.fulfilledProcessor);
-        this.thread.start();
-        
-        Thread.sleep(MIN_SLEEP_TIME);
-
-        Assert.assertEquals(this.connectionAttempts.size(), 0);
-    }
-    
+    // test case: When running thread in FulfilledProcessor without the external addresses of the
+    // reverse tunnel, the method processFulfilledOrder() must not change OrderState to Failed and
+    // must remain in Fulfilled list.
+    @SuppressWarnings("static-access")
```

Comment:
If you agree with me on line 124, this line is no longer necessary, right?

Reply:
This line is added by of Eclipse IDE recommendation...
It will be removed.